presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestHiveQueries.java

```diff
@@ -45,14 +45,14 @@ protected QueryRunner createQueryRunner()
         assertNotNull(prestoServerPath, "Native worker binary path is missing. Add -DPRESTO_SERVER=<path/to/presto_server> to your JVM arguments.");
         assertNotNull(dataDirectory, "Data directory path is missing. Add -DDATA_DIR=<path/to/data> to your JVM arguments.");
 
-        return HiveExternalWorkerQueryRunner.createNativeQueryRunner(dataDirectory, prestoServerPath, Optional.ofNullable(workerCount).map(Integer::parseInt), cacheMaxSize, useThrift);
+        return HiveExternalWorkerQueryQueryRunner.createNativeQueryRunner(dataDirectory, prestoServerPath, Optional.ofNullable(workerCount).map(Integer::parseInt), cacheMaxSize, useThrift);
```

Comment:
Curious why it's called `HiveExternalWorkerQueryQueryRunner` instead of `HiveExternalWorkerQueryRunner` ?

Reply:
Those are some inter-commit refactoring errors. I have corrected them. Thanks!